showcase/pom.xml

```diff
@@ -63,6 +74,61 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-prometheus</artifactId>
+      <version>1.27.0-alpha</version>
+    </dependency>
+    <!--Used by Spring actuator-->
+    <dependency>
+      <groupId>io.micrometer</groupId>
+      <artifactId>micrometer-registry-prometheus</artifactId>
+      <version>1.9.7</version>
+    </dependency>
```

Comment:
I might have missed this above, but where are we using this inside showcase?

Reply:
not using it, forgot to remove. now removed. thank you.